storage/src/main/scala/coop/rchain/storage/Bb.scala

```diff
@@ -0,0 +1,50 @@
+/*               __          __                                         *\
+**    __________/ /_  ____ _/_/___                                      **
+**   / ___/ ___/ __ \/ __ `/ / __ \     RChain API                      **
+**  / /  / /__/ / / / /_/ / / / / /     (c) http://rchain.coop          **
+** /_/   \___/_/ /_/\____/_/_/ /_/                                      **
+\*                                                                      */
+
+// Bb wraps Java ByteBuffer and adds type genericity.
+
+package coop.rchain.storage
+
+import java.nio.ByteBuffer
+import java.nio.ByteBuffer.allocateDirect
+import java.nio.charset.StandardCharsets.UTF_8
+
+object Bb {
+  trait Bbable[T]
+  object Bbable {
+    implicit object ShortEv extends Bbable[Short]
```

Comment:
Looks quite interesting for me. Is this solution somehow differs from type tagging?